operator/cnp_event.go

```diff
@@ -16,15 +16,22 @@ package main
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
I only read the commit description here, probably it makes sense to have the 10s as a flag for cilium operator? :thinking: